docs/build/groups.md

```diff
@@ -0,0 +1,618 @@
+
+# Manage group chats with XMTP
+
+![Status](https://img.shields.io/badge/Reference_implementation_status-Alpha-orange)
+
+Secure group chats are an important part of every messaging app. In this guide, we delve into the essentials of using XMTP for creating secure group chats. From the initial steps of starting a new group chat, listing and caching conversations for quick access, to advanced topics like managing group members and synchronizing message history data across devices.
+
+:::caution Group Chats are Per Installation
+Group chats in XMTP are specific to each installation. This means that while you will see your group chat conversations across different devices, you will not see the historical messages within those chats automatically. Each group chat's message history is tied to the device where it was started. Consequently, there is no message history synced across devices. When you sign in on a new device, you will be able to see existing group chat conversations but will only receive new messages going forward. This behavior is specific to group conversations.
+:::
+
+## Create a group chat
+
+Initiate a new group chat with a list of specified addresses. To create a group, the recipient must have already started their client at least once on the XMTP network.
+
+<Tabs groupId="groupchats">
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+```jsx
+const group = await client.conversations.newGroup([
+  walletAddress1,
+  walletAddress2,
+]);
+```
+
+</TabItem>
+<TabItem value="kotlin" label="Kotlin" attributes={{className: "kotlin_tab"}}>
+
+```kotlin
+val group = client.conversations.newGroup(listOf(walletAddress1,walletAddress2))
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift"  attributes={{className: "swift_tab"}}>
+
+```swift
+let group = try await client.conversations.newGroup(with: [walletAddress1, walletAddress2])
+```
+
+</TabItem>
+<TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
+
+Code sample expected for Q2 2024
+
+</TabItem>
+<TabItem value="js" label="JavaScript"  attributes={{className: "js_tab "}}>
+
+Code sample expected for Q2 2024
+
+</TabItem>
+<TabItem value="react" label="React"  attributes={{className: "react_tab "}}>
+
+Code sample expected for Q2 2024
+
+</TabItem>
+</Tabs>
+
+_The maximum amount of addresses allowed is 250._
+
+## List group chat conversations
+
+Retrieve all existing group chat conversations associated with the current XMTP client. Refer to the [Conversations](/docs/build/conversations.md) section for more details.
+
+<Tabs groupId="groupchats">
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+```jsx
+const groups = await client.conversations.listGroups();
+```
+
+</TabItem>
+<TabItem value="kotlin" label="Kotlin" attributes={{className: "kotlin_tab"}}>
+
+```kotlin
+// List all group chats for the current user
+val groups = client.conversations.listGroups()
```

Comment:
Might be a good place to call out client.conversations.syncGroups() since the list groups will just pull whatever is in the database. So if you want the latest sync before listing to kick off a network request for the latest groups.